ldcoolp/config/default.ini

```diff
@@ -48,6 +48,8 @@ readme_template = README_template.md
 log_parent_dir = %(parent_dir)s
 log_dir = logs
 
+# Footer to strip
+footer = </p><p><br></p><p><br></p><p></p><hr><p></p><p><br></p><p><i>For inquiries regarding
```

Comment:
Is this *exact* string always inserted as the description footer?

I saw a:

```text
<p><i>For
inquiries regarding the contents of this dataset, please contact the
Corresponding Author listed in the README.txt file. Administrative inquiries
(e.g., removal requests, trouble downloading, etc.) can be directed to data-management@arizona.edu</i></p><br></div>
```
in deposit 11977818

Reply:
It doesn't have to be the full footer. Just enough of the beginning. Partition will strip out anything afterwards.

Reply:
Hmm, I tried running it on a previous dataset, 12026436 and it didn't partition it. Though this is one of the earlier deposits. I'll experiment with others.

Reply:
Additional work was done and I believe we cover a wider domain of edge cases